pyluos/version.py

```diff
@@ -1 +1 @@
-version = '1.1.0'
+version = '1.1.1'
```

Comment:
We gonna start doing pre-release following a `+n` rule. Can you put `1.1.0+1`instead of `1.1.1`